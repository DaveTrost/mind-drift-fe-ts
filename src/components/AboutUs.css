.logo {
  margin-top: 3em;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  text-align: center;
  p {
    font-family: Damion;
    font-size: 1em;
    color: var(--peach);
    word-wrap: break-word;
  }
  img {
    max-width: 200vw;
    height: auto;
  }
  .blurb {
    width: 65%;
    font-size: 1.5em;
  }
}

.AboutUs {
  display: flex;
  flex-direction: column;
  margin-bottom: 4em;
  margin-left: 2em;
  margin-right: 2em;
  h1 {
    font-family: Montserrat;
    font-weight: 700;
    font-size: 1.5em;
    text-transform: uppercase;
    margin-top: 1em;
    margin-bottom: 1em;
    color: var(--dark-blue);
  }
  h2 {
    font-family: Damion;
    font-size: 1.75em;
    color: var(--dark-blue);
    margin-top: 0.5em;
  }
  img {
    border-radius: 50%;
    width: 100px;
    height: auto;
    border: 4px solid var(--light-blue);
  }
  p {
    font-family: Montserrat;
    font-weight: 200;
    margin-bottom: 1.5em;
    color: var(--dark-blue);
  }
}
