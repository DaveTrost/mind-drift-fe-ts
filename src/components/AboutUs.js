import React from 'react';
import styles from './AboutUs.css';

const AboutUs () => {
  return (
    <section className={styles.AboutUs}>

    </section>
  );
}

export default AboutUs;