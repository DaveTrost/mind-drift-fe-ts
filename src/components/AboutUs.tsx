import * as React from 'react';
import { useSpring, animated } from 'react-spring';
import NavBar from './NavBar';

const styles = require('./AboutUs.css');
const brownFlowerIcon = require('../assets/brownFlowerIcon.png');
const abbeyImg = require('../assets/ABBEY.jpg');
const daveImg = require('../assets/DAVE.png');
const maeveImg = require('../assets/MAEVE.jpg');

const AboutUs = () => {
  const spring = useSpring({
    opacity: 1,
    from: { opacity: 0.1 },
    config: { duration: 1500 },
  });

  return (
    <>
      <NavBar />
      <animated.div
        className={styles.logo}
        style={spring}>
        <img src={brownFlowerIcon} />
        <p className={styles.blurb}>mindDrift is a mindful breathing app, aimed to make your experience seamless and convenient.</p>
        <section className={styles.AboutUs}>
          <h1>Meet Our Team:</h1>
          <div>
            <img src={abbeyImg} alt="Abbey Masters" />
            <h2>Abbey Masters</h2>
            <p>Abbey is a Full-Stack developer, and enjoys long walks at the park with her dog. And also chocolate chip cookies!</p>
          </div>
          <div>
            <img src={daveImg} alt="Dave Trost" />
            <h2>Dave Trost</h2>
            <p>Dave is a Back-End developer who is surprised by how much he likes React and Redux. He is a big nerd for disc golf, and lots of other things too. Lots of GF chocolate chip cookies!</p>
          </div>
          <div>
            <img src={maeveImg} alt="Maeve Griffin" />
            <h2>Maeve Griffin</h2>
            <p>Maeve is a Full-Stack Javascript developer who has learned a lot about React Spring.  And likes trying new chocolate chip cookie recipes.</p>
          </div>
        </section>
      </animated.div>
    </>
  );
};

export default AboutUs;
