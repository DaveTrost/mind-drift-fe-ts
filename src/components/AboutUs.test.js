import React from 'react';
import { shallow } from 'enzyme';
import AboutUs from './AboutUs';

describe('About Us component', () => {
  it('renders', () => {
    Date.now = () => 42;
    const wrapper = shallow(<AboutUs />);
    expect(wrapper).toMatchSnapshot();
  });
});
