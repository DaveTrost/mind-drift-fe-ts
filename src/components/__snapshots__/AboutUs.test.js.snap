// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Us component renders 1`] = `
<Fragment>
  <NavBar />
  <ForwardRef
    className="logo"
    style={
      Object {
        "opacity": AnimatedValue {
          "animatedStyles": Set {},
          "children": Array [],
          "done": false,
          "lastPosition": 0.1,
          "lastTime": undefined,
          "lastVelocity": undefined,
          "payload": undefined,
          "setValue": [Function],
          "startPosition": 0.1,
          "startTime": 42,
          "value": 0.1,
        },
      }
    }
  >
    <img
      src={
        Object {
          "default": "test-file-stub",
        }
      }
    />
    <p
      className="blurb"
    >
      mindDrift is a mindful breathing app, aimed to make your experience seamless and convenient.
    </p>
    <section
      className="AboutUs"
    >
      <h1>
        Meet Our Team:
      </h1>
      <div>
        <img
          alt="Abbey Masters"
          src={
            Object {
              "default": "test-file-stub",
            }
          }
        />
        <h2>
          Abbey Masters
        </h2>
        <p>
          Abbey is a Full-Stack developer, and enjoys long walks at the park with her dog. And also chocolate chip cookies!
        </p>
      </div>
      <div>
        <img
          alt="Dave Trost"
          src={
            Object {
              "default": "test-file-stub",
            }
          }
        />
        <h2>
          Dave Trost
        </h2>
        <p>
          Dave is a Back-End developer who is surprised by how much he likes React and Redux. He is a big nerd for disc golf, and lots of other things too. Lots of GF chocolate chip cookies!
        </p>
      </div>
      <div>
        <img
          alt="Maeve Griffin"
          src={
            Object {
              "default": "test-file-stub",
            }
          }
        />
        <h2>
          Maeve Griffin
        </h2>
        <p>
          Maeve is a Full-Stack Javascript developer who has learned a lot about React Spring.  And likes trying new chocolate chip cookie recipes.
        </p>
      </div>
    </section>
  </ForwardRef>
</Fragment>
`;
